### DISCUSSION TITLE
Is AWS Redshift supported?

### DISCUSSION THREAD

**Discussion by saeed-abdul-rahim** [fastapi-users/fastapi-users#938] [General]
In documentation,  asyncio sqls are required.
Is Redshift supported?

**Comment by frankie567** [comment]
No, it's not. However, thanks to database adapters, it's very easy to add support for any backend in your project. All you have to do is to create a class inheriting from [BaseUserDatabase ](https://github.com/fastapi-users/fastapi-users/blob/master/fastapi_users/db/base.py)and implement the methods.

You can take inspiration from the [MongoDB ](https://github.com/fastapi-users/fastapi-users-db-mongodb) one or the [SQLALchemy](https://github.com/fastapi-users/fastapi-users-db-sqlalchemy) one.
